files/files.go

```diff
@@ -316,14 +328,25 @@ func GetBaseName(filename string) string {
 
 // Getter gets a directory or file using the Hashicorp go-getter library
 // See https://github.com/hashicorp/go-getter
-func Getter(url, dst string) error {
+func Getter(source, dst string) error {
+
+	//	Validate the URL first.
+	url, err := url.ParseRequestURI(source)
```

Comment:
Why do we need to validate the URL first? Are we sure that url.ParseRequestURI will not return an error for all URLS/URI's supported by go-getter ?

Reply:
I think you are right. I added the URI parser primarily for branching the code to handle files and URLs separately. But then I forgot I shifted it to `ResolveFile()` function. I'm pushing the change.